### DISCUSSION TITLE
http/quic-performance

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#171] [General]
# http/quic-performance

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/http/quic-performance.html

**Comment by sanbei101** [comment]
原来普及这么慢是有原因的

**Reply by isno** [reply]
我在第一章的小节里面写了这么一段话：从根本上讲，驱动技术发展的动力有很多种，但能让技术大行其道的动力，都是来源于业务发展的需要。

QUIC 的话，背后的驱动力没那么强劲。HTTP/2 稳定的情况下，没人愿意去“趟雷”。
